ocean_lib/common/aquarius/aquarius.py

```diff
@@ -135,7 +135,7 @@ def text_search(self, text, sort=None, offset=100, page=1):
         if response.status_code == 200:
             return self._parse_search_response(response.content)
         else:
-            raise Exception(f"Unable to search for DDO: {response.content}")
+            raise ValueError(f"Unable to search for DDO: {response.content}")
```

Comment:
the code diff is failing because the tests are still checking for a generic exception. Take care to adapt the tests too! Instead of catching an Exception, you need to catch a Value Error now, where this is tested.

Reply:
Should I keep it like that, or revert to `Exception`?